lib/Mojo/Promise.pm

```diff
@@ -290,6 +316,33 @@ reason.
     say "We are done!";
   });
 
+=head2 map
+
+  my $promise = Mojo::Promise->map(sub { ... }, @items);
+  my $promise = Mojo::Promise->map({concurrency => 3}, sub { ... }, @items);
+
+Apply a function that returns a L<Mojo::Promise> to each item in a list of
+items while optionally limiting concurrency. Returns a L<Mojo::Promise> that
+collects the results in the same manner as L</all>.
+
+Note that if any item's promise is rejected, any remaining items which have not
+yet been mapped will not be.
+
+  Mojo::Promise->map({concurrency => 3}, sub { $ua->get_p($_) }, @urls)
+    ->then(sub{ say $_->[0]->res->dom->at('title')->text for @_ });
```

Comment:
I think there’s a tiny glitch in the documentation here.

Reply:
I'm not in a good position to check at the moment, but visually I don't see a glitch. What do you see @jhthorsen ?

Reply:
“sub{“ - missing a space.